src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/EnvelopeReference.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents BulkScanEnvelope complex type in CCD case definitions.
+ */
+public class EnvelopeReference {
+
+    @JsonProperty
+    public String id;
+
+    @JsonProperty
+    public String action;
+
+    public EnvelopeReference(
+        @JsonProperty("id") String id,
+        @JsonProperty("action") String action
```

Comment:
are those required? should we even start marking as required since we rarely used that feature before?

Reply:
Both are required, actually. I'll add that.

Reply:
done